python/sglang/srt/layers/attention/flashinfer_backend.py

```diff
@@ -313,7 +314,7 @@ def init_forward_metadata_capture_cuda_graph(
                         paged_kv_indices_buf=self.cuda_graph_kv_indices[i],
                         paged_kv_last_page_len_buf=self.kv_last_page_len[:bs],
                         custom_mask_buf=self.cuda_graph_custom_mask,
-                        qk_indptr_buf=self.cuda_graph_qk_indptr[i][: bs + 1],
+                        mask_indptr_buf=self.cuda_graph_qk_indptr[i][: bs + 1],
```

Comment:
This is a breaking change from flashinfer @yzh119: the parameter name has been changed.